local/projects/configured.go

```diff
@@ -40,3 +46,25 @@ func GetConfiguredAndRunning(proxyProjects, runningProjects map[string]*Configur
 	}
 	return projects, nil
 }
+
+func GetConfiguredAndRunningProjectsDirectories(configuredAndRunning map[string]*ConfiguredProject) ([]string, error) {
+	var dirs []string
+
+	for dir := range configuredAndRunning {
+		// Go does not automatically expand ~ in paths
+		if strings.HasPrefix(dir, "~") {
+			homeDir, err := homedir.Dir()
+			if err != nil {
+				return nil, err
+			}
+
+			dir = strings.Replace(dir, "~", homeDir, 1)
+		}
```

Comment:
expanding ~ whereas we are the ones replacing the absolute path with it seems a bit silly, maybe we can refactor `ToConfiguredProjects` or `projects.GetConfiguredAndRunning` instead?

Reply:
That's right, I added a flag to `ToConfiguredProjects` 👍

Reply:
thank you